integration_tests/suite/test_sync_db.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2025 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from .helpers.base import (
+    MASTER_TENANT,
+    MASTER_TOKEN,
+    OTHER_TENANT,
+    OTHER_USER_UUID,
+    USER_1_UUID,
+    USER_2_UUID,
+    USERS_TENANT,
+    WAZO_UUID,
+    BaseIntegrationTest,
+)
+from .helpers.fixtures import subscription
+from .helpers.wait_strategy import EverythingOkWaitStrategy
+
+USER_1_TEST_SUBSCRIPTION = {
+    'name': 'test',
+    'service': 'http',
+    'config': {'url': 'http://test.example.com', 'method': 'get'},
+    'events': ['test'],
+    'owner_user_uuid': USER_1_UUID,
+    'events_user_uuid': USER_1_UUID,
+    'events_wazo_uuid': WAZO_UUID,
+}
+
+USER_2_TEST_SUBSCRIPTION = {
+    'name': 'test',
+    'service': 'http',
+    'config': {'url': 'http://test.example.com', 'method': 'get'},
+    'events': ['test'],
+    'owner_user_uuid': USER_2_UUID,
+    'events_user_uuid': USER_2_UUID,
+    'events_wazo_uuid': WAZO_UUID,
+}
+
+USER_2_TEST_SUBSCRIPTION_WATCH_USER_1 = {
+    'name': 'test',
+    'service': 'http',
+    'config': {'url': 'http://test.example.com', 'method': 'get'},
+    'events': ['test'],
+    'owner_user_uuid': USER_2_UUID,
+    'events_user_uuid': USER_1_UUID,
+    'events_wazo_uuid': WAZO_UUID,
+}
+
+OTHER_USER_TEST_SUBSCRIPTION = {
+    'name': 'test',
+    'service': 'http',
+    'config': {'url': 'http://test.example.com', 'method': 'get'},
+    'events': ['test'],
+    'owner_user_uuid': OTHER_USER_UUID,
+    'owner_tenant_uuid': OTHER_TENANT,
+    'events_user_uuid': OTHER_USER_UUID,
+    'events_wazo_uuid': WAZO_UUID,
+}
+
+
+class TestSyncDB(BaseIntegrationTest):
+    asset = 'base'
+    wait_strategy = EverythingOkWaitStrategy()
+
+    def setUp(self):
+        super().setUp()
+
+    @classmethod
+    def sync_db(cls):
+        print(cls.docker_exec(['wazo-webhookd-sync-db', '--debug']))
```

Comment:
Remaining debug print?
If we really want this info, we should print it only on fail

Reply:
Thanks, done